optimism.tokenlist.json

```diff
@@ -891,7 +891,40 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
-	}
+	},
+    {
+      "chainId": 1,
+      "address": "0x3af33bef05c2dcb3c7288b77fe1c8d2aeba4d789",
+      "name": "Kromatika",
+      "symbol": "KROM",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/KROM.png",
```

Comment:
You might want to drop the size of the logo to something closer to 256x256 as this might not display properly.

Reply:
dropped in the latest commit to 256/256